docs/DeveloperGuide.md

```diff
@@ -100,11 +106,11 @@ This section will describe how the main features of the application are implemen
  2. A <code>Parser</code> object is created to call its <code>parseCommand</code> method.
      * The <code>Parser</code> object instantiates an <code>AddCommand</code> object based on the user input.
  3. The <code>Duke</code> class calls the <code>AddCommand#execute()</code> method of the <code>AddCommand</code> object.
- 4. In the <code>AddCommand#execute()</code> function, the <code>item</code> to be added is stored in the <code>ShoppingList</code> 
-    object, using items.add().
- 5. In the sequence diagram, the AddCommand will add <code>item</code> if the description is provided and one / both price and 
+ 4. In the <code>AddCommand#execute()</code> function, the <code>Item</code> to be added is stored in the <code>ShoppingList</code> 
+    object, using <code>Item.add()</code>.
```

Comment:
It seems the method referred here isn't 'Item.add()', but rather 'ShoppingList#add()'

Reply:
Oh yeah thanks for catching it! Will make the changes.